formats/gltf/extensions.go

```diff
@@ -447,8 +460,49 @@ func (pd PolyformDispersion) ExtensionID() string {
 	return "KHR_materials_dispersion"
 }
 
-func (pd PolyformDispersion) ToExtensionData(w *Writer) map[string]any {
+func (pd PolyformDispersion) ToMaterialExtensionData(w *Writer) map[string]any {
 	metadata := make(map[string]any)
 	metadata["dispersion"] = pd.Dispersion
 	return metadata
 }
+
+// KHR_texture_transform ===================================================
+// https://github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Khronos/KHR_texture_transform
+
+var _ TextureExtension = PolyformTextureTransform{}
```

Comment:
This is a compile-time and IDE check that the type fulfills the interface. 
The tests should check that as well, but this will be visible even without running the tests, handy in development.

Reply:
It's not really doing anything beyond a small developer convenience, I'm happy to remove it.